v3/src/components/graph/adornments/count/count-adornment-component.tsx

```diff
@@ -109,21 +130,22 @@ export const CountAdornment = observer(function CountAdornment(props: IAdornment
     return mstAutorun(
       () => {
         getAxisDomains(xAxis, yAxis)
-        subPlotRegionBoundariesRef.current = adornmentsStore?.subPlotRegionBoundaries(instanceKey, scale) ?? []
+        subPlotRegionBoundariesRef.current = subPlotRegionBoundaries()
         plotCaseCounts()
       }, { name: "Count.refreshBoundariesAndCaseCounts" }, model)
-  }, [adornmentsStore, instanceKey, model, plotCaseCounts, plotHeight, plotWidth, scale, xAxis, yAxis])
+  }, [model, plotCaseCounts, subPlotRegionBoundaries, xAxis, yAxis])
 
   useEffect(function refreshShowPercentOption() {
     return mstAutorun(
       () => {
         // set showPercent to false if attributes change to a configuration that doesn't support percent
-        const shouldShowPercentOption = !!dataConfig?.categoricalAttrCount || adornmentsStore.subPlotsHaveRegions
+        const shouldShowPercentOption = !!dataConfig?.categoricalAttrCount || adornmentsStore.subPlotsHaveRegions ||
+                                        graphModel.pointDisplayType === "bins"
         if (!shouldShowPercentOption && model?.showPercent) {
           model.setShowPercent(false)
         }
      }, { name: "CountAdornment.refreshPercentOption"}, model)
-  }, [adornmentsStore.subPlotsHaveRegions, dataConfig, model])
+  }, [adornmentsStore.subPlotsHaveRegions, dataConfig, graphModel.pointDisplayType, model])
```

Comment:
```suggestion
  }, [adornmentsStore, dataConfig, graphModel, model])
```
If the `subPlotsHaveRegions` or `pointDisplayType` properties change we want the `autorun` to trigger rather than being reinstalled.